app/controllers/stats_controller.rb

```diff
@@ -0,0 +1,38 @@
+class StatsController < ApplicationController
+  include ActionController::HttpAuthentication::Token::ControllerMethods
+
+  before_action :authenticate, :set_access_control_headers
```

Comment:
As mentioned on the Trello card: when we discussed the actions for this card we agreed that it didn't need to begin with authentication as it could solve the need of basic statistic counts without the extra wrapper of complexity and barrier to people accessing these numbers (especially within the department). Did something change since then?